ui/styles/theme.go

```diff
@@ -1,40 +1,68 @@
-	SubleMainText      lipgloss.AdaptiveColor
+	SelectedBackground lipgloss.AdaptiveColor // config.Theme.Colors.Background.Selected
+	Border             lipgloss.AdaptiveColor // config.Theme.Colors.Border.Primary
+	FaintBorder        lipgloss.AdaptiveColor // config.Theme.Colors.Border.Faint
+	SecondaryBorder    lipgloss.AdaptiveColor // config.Theme.Colors.Border.Secondary
+	FaintText          lipgloss.AdaptiveColor // config.Theme.Colors.Text.Faint
+	MainText           lipgloss.AdaptiveColor // config.Theme.Colors.Text.Primary
+	SecondaryText      lipgloss.AdaptiveColor // config.Theme.Colors.Text.Secondary
+	SubleMainText      lipgloss.AdaptiveColor // config.Theme.Colors.Text.Inverted
+	SuccessText        lipgloss.AdaptiveColor // config.Theme.Colors.Text.Success
+	WarningText        lipgloss.AdaptiveColor // config.Theme.Colors.Text.Warning
 }
 
-var subtleIndigo = lipgloss.AdaptiveColor{Light: "#5A57B5", Dark: "#242347"}
-
-var DefaultTheme = Theme{
-	MainText:           lipgloss.AdaptiveColor{Light: "#242347", Dark: "#E2E1ED"},
-	BrightMainText:     lipgloss.AdaptiveColor{Light: "#242347", Dark: "#E2E1ED"},
-	SubleMainText:      subtleIndigo,
-	Border:             lipgloss.AdaptiveColor{Light: indigo.Light, Dark: indigo.Dark},
-	SecondaryBorder:    lipgloss.AdaptiveColor{Light: indigo.Light, Dark: "#39386b"},
-	WarningText:        lipgloss.AdaptiveColor{Light: "#F23D5C", Dark: "#F23D5C"},
-	SuccessText:        lipgloss.AdaptiveColor{Light: "#3DF294", Dark: "#3DF294"},
-	FaintBorder:        lipgloss.AdaptiveColor{Light: "#D9DCCF", Dark: "#2b2b40"},
-	FaintText:          lipgloss.AdaptiveColor{Light: indigo.Light, Dark: "#3E4057"},
-	SelectedBackground: lipgloss.AdaptiveColor{Light: subtleIndigo.Light, Dark: "#39386b"},
-	SecondaryText:      lipgloss.AdaptiveColor{Light: indigo.Light, Dark: "#666CA6"},
-}
+var theme *Theme
 
-var (
-	SingleRuneWidth    = 4
-	MainContentPadding = 1
-)
+var DefaultTheme = func() Theme {
+	if theme != nil {
+		return *theme
+	}
+
+	_shimHex := func(hex config.HexColor) lipgloss.AdaptiveColor {
+		return lipgloss.AdaptiveColor{Light: string(hex), Dark: string(hex)}
+	}
+
+	cfg, _ := config.ParseConfig()
+
+	if cfg.Theme == nil {
+		theme = &Theme{
+			SelectedBackground: lipgloss.AdaptiveColor{Light: "006", Dark: "006"},
+			Border:             lipgloss.AdaptiveColor{Light: "000", Dark: "015"},
+			FaintBorder:        lipgloss.AdaptiveColor{Light: "007", Dark: "008"},
+			SecondaryBorder:    lipgloss.AdaptiveColor{Light: "008", Dark: "007"},
+			FaintText:          lipgloss.AdaptiveColor{Light: "243", Dark: "249"},
+			MainText:           lipgloss.AdaptiveColor{Light: "000", Dark: "015"},
+			SecondaryText:      lipgloss.AdaptiveColor{Light: "237", Dark: "255"},
+			SubleMainText:      lipgloss.AdaptiveColor{Light: "015", Dark: "015"},
+			SuccessText:        lipgloss.AdaptiveColor{Light: "002", Dark: "002"},
+			WarningText:        lipgloss.AdaptiveColor{Light: "001", Dark: "001"},
```

Comment:
I played around with the colors on a dark theme and light theme terminals and I think I like these colors better:
```go
			SelectedBackground: lipgloss.AdaptiveColor{Light: "006", Dark: "008"},
			Border:             lipgloss.AdaptiveColor{Light: "013", Dark: "008"},
			FaintBorder:        lipgloss.AdaptiveColor{Light: "254", Dark: "000"},
			SecondaryBorder:    lipgloss.AdaptiveColor{Light: "008", Dark: "007"},
			FaintText:          lipgloss.AdaptiveColor{Light: "007", Dark: "249"},
			MainText:           lipgloss.AdaptiveColor{Light: "000", Dark: "015"},
			SecondaryText:      lipgloss.AdaptiveColor{Light: "244", Dark: "251"},
			SubleMainText:      lipgloss.AdaptiveColor{Light: "015", Dark: "236"},
			SuccessText:        lipgloss.AdaptiveColor{Light: "002", Dark: "002"},
			WarningText:        lipgloss.AdaptiveColor{Light: "001", Dark: "001"},
```
Let me know what you think!

Reply:
Works well on my default shell and Kitty, but on Konqueror it's somewhat borked. I'm not really passionate about this because I can make my own theme if I don't like the default now ;)

![image](https://user-images.githubusercontent.com/34752364/179360834-1cbf338a-9365-4ab7-8511-f9682e333e8d.png)
![image](https://user-images.githubusercontent.com/34752364/179360844-435ef925-b0f5-4ef6-80e2-0944aca6d85a.png)

Reply:
Cool then! I'll change the ansi colors to mine as I prefer the borders not to be high contrast.
But I too will just use my custom theme :)